packages/alice_objectbox/lib/core/alice_core_objectbox.dart

```diff
@@ -0,0 +1,181 @@
+import 'dart:async';
+import 'dart:math' show max;
+
+import 'package:alice/core/alice_core.dart';
+import 'package:alice/core/alice_utils.dart';
+import 'package:alice/helper/alice_save_helper.dart';
+import 'package:alice/model/alice_http_call.dart';
+import 'package:alice/model/alice_http_error.dart';
+import 'package:alice/model/alice_http_response.dart';
+import 'package:alice_objectbox/alice_store.dart';
+import 'package:alice_objectbox/model/cached_alice_http_call.dart';
+import 'package:alice_objectbox/model/cached_alice_http_error.dart';
+import 'package:alice_objectbox/objectbox.g.dart';
+import 'package:flutter/widgets.dart';
+
+class AliceCoreObjectBox extends AliceCore {
+  AliceCoreObjectBox(
+    super.navigatorKey, {
+    required AliceStore store,
+    required super.showNotification,
+    required super.showInspectorOnShake,
+    required super.notificationIcon,
+    required super.maxCallsCount,
+    super.directionality,
+    super.showShareButton,
+  }) : _store = store;
+
+  final AliceStore _store;
```

Comment:
Can we clean all records after startup?

Reply:
Hm, I could add an additional parameter to the constructor, however, this is persistent storage so it also kinda defies the point, don't you think?. 🤷

Reply:
Added `bool empty` to the `AliceStore.create` method.